src/database/schema/UserSchema.ts

```diff
@@ -92,8 +97,9 @@ class UserSchema implements BaseDbSchema {
         anon_user_info_emoji_name,
         anon_user_info_emoji_code,
         anon_user_info_color_name,
-        anon_user_info_color_code
-      ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);`;
+        anon_user_info_color_code,
+        is_anonymous
+      ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `is_anonymous` field is being inserted into the database without any null check. If `isAnonymous` is null, it might cause issues. Please ensure that you have a default value or handle null values appropriately.

```diff
-        anon_user_info_color_code,
-        is_anonymous
+        anon_user_info_color_code,
+        is_anonymous || IS_NOT_ANONYMOUS
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->